content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -234,6 +234,17 @@ This limitation surfaces in multi-socket, as well as single-socket multi NUMA sy
 and can cause significant performance degradation in latency-critical execution and high-throughput applications if the
 Topology Manager decides to align resources on non-adjacent NUMA nodes.
 
+* `max-allowable-numa-nodes` (beta, visible by default).
+The `max-allowable-numa-nodes` policy option is beta in Kubernetes {{< skew currentVersion >}}.
+
+With the `TopologyManager` enabled, the time to admit a pod is tied to the number of NUMA nodes on the physical machine.
+With the new `max-allowable-numa-nodes` option, nodes with more than 8 NUMA nodes can be alowed to run
```

Comment:
```suggestion
With the new `max-allowable-numa-nodes` option, nodes with more than 8 NUMA nodes can be allowed to run
```

Reply:
We also (strongly) avoid “new“; we like documentation to be [timeless](https://developers.google.com/style/timeless-documentation).

I'd make an exception for a page that was entirely about an alpha feature, because the quality bar is lower for alpha docs.